src/jrd/Monitoring.cpp

```diff
@@ -549,6 +549,12 @@ MonitoringSnapshot::MonitoringSnapshot(thread_db* tdbb, MemoryPool& pool)
 
 	// Parse the dump
 
+	// BlobID's of statement text and plan
+	using StmtBlobs = struct { bid text; bid plan; };
```

Comment:
Why invent new way to do a simple thing?
```suggestion
	struct StmtBlobs { bid text; bid plan; };
```